Content.Client/Chemistry/UI/TransferAmountBoundUserInterface.cs

```diff
@@ -24,7 +24,7 @@ protected override void Open()
                     _window.Close();
                 }
             };
-
+            _window.OnClose += Close;
```

Comment:
As far as I can tall, this line not being there was causing issues with this window. Without this, the bound user interface was never properly closed.

The fact that `(...).OnClose += Close;` is duplicated in just about every bound user interface and leads to bugs when omitted is probably not great.

Reply:
bound UIs have a lot of boilerplate, there's been some discussion on how to fix that but nothing concrete yet, my bad for this code specifically 😎